conf/layer.conf

```diff
@@ -19,6 +19,7 @@ BBFILE_PATTERN_openrc-layer := "^${LAYERDIR}/"
 
 BBFILES_DYNAMIC += " \
     networking-layer:${LAYERDIR}/dynamic-layers/meta-networking/*/*.bbappend \
+    oe-layer:${LAYERDIR}/dynamic-layers/meta-oe/*/*.bbappend \
```

Comment:
This needs to be `openembedded-layer`, not `oe-layer`.  You can tell from what the layer adds to `BBFILE_COLLECTIONS` (http://cgit.openembedded.org/meta-openembedded/tree/meta-oe/conf/layer.conf#n17).

For testing that it's getting correctly applied, you can do something like `bitbake -f -c package modemmanager` and then look in your `TMPDIR` (`cat tmp/build-glibc/work/core2-64-oe-linux/modemmanager/1.18.10-r0/packages-split/modemmanager/etc/init.d/modemmanager` for me).

Reply:
Though, honestly, for testing you should really be starting up an image with these scripts and running them to make sure they're working.

Reply:
> This needs to be `openembedded-layer`, not `oe-layer`. You can tell from what the layer adds to `BBFILE_COLLECTIONS` (http://cgit.openembedded.org/meta-openembedded/tree/meta-oe/conf/layer.conf#n17).
> 
> For testing that it's getting correctly applied, you can do something like `bitbake -f -c package modemmanager` and then look in your `TMPDIR` (`cat tmp/build-glibc/work/core2-64-oe-linux/modemmanager/1.18.10-r0/packages-split/modemmanager/etc/init.d/modemmanager` for me).

There it is! Couldn't find that in the moment of preparing this, thanks!